apps/main/src/llamalend/features/market-position-details/HealthBar.tsx

```diff
@@ -99,10 +158,9 @@ export const HealthBar = ({ health, softLiquidation, small }: HealthBarProps) =>
             transition: 'width 0.3s ease-in-out, background-color 0.3s ease-in-out',
           }}
         />
-        <Line first position="0%" color={'red'} />
-        <Line position="15%" color={'orange'} />
-        <Line position="50%" color={'green'} />
-        <Line position="100%" color={'dark-green'} />
+        {lines.map((line, index) => (
+          <Line key={`${line.position}-${index}`} position={line.position} color={line.color} />
```

Comment:
Do you expect duplicates here? Otherwise, better leave the index out of the key
```suggestion
          <Line key={line.position} position={line.position} color={line.color} />
```